taiga_halo2/src/circuit/vp_circuit.rs

```diff
@@ -32,6 +32,14 @@ impl VPVerifyingInfo {
         let params = SETUP_PARAMS_MAP.get(&VP_CIRCUIT_PARAMS_SIZE).unwrap();
         self.proof.verify(&self.vk, params, &[&self.instance])
     }
+
+    pub fn get_nullifiers(&self) -> [pallas::Base; NUM_NOTE] {
+        [self.instance[0], self.instance[2]]
```

Comment:
It looks a bit clunky. What's the logic behind this indexes? Would it be possible to make it more declarative? Can it go wrong (e.g. by instances not having at least for elements or by having elements in the wrong position)?

Reply:
I'll add the constant index names to make it more declarative